clients/go/gengo.go

```diff
@@ -243,7 +243,7 @@ func doNothing(baseTransport http.RoundTripper, spanNameCtxValue interface{}, tp
 func determineSampling() (samplingProbability float64, err error) {
 
 		// If we're running locally, then turn off sampling. Otherwise sample
-		// 1% or whatever TRACING_SAMPLING_PROBABILITY specifies.
+		// 1%% or whatever TRACING_SAMPLING_PROBABILITY specifies.
```

Comment:
Does this get parsed by package `fmt`?

Reply:
Ah I just read the previous PR, so I understand these changes. Still curious about what parses this string?

Reply:
It's actually parsed by template. 

https://github.com/Clever/wag/blob/7904b872303587fb952883c24f27aca2e8d1dc39/clients/go/gengo.go#L450-L451

Reply:
Interesting. I didn't realize template uses replacement the way `fmt` does.

Reply:
Neither did I until I saw (MISSING) showing up in my comments.